### DISCUSSION TITLE
Welcome to stable-diffusion-webui/ux  Discussions!

### DISCUSSION THREAD

**Discussion by anapnoe** [anapnoe/stable-diffusion-webui-ux#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nacurutu** [comment]
Perfect, better place to discuss about this PR... 👍🏻

**Comment by ghost** [comment]


**Reply by anapnoe** [reply]
please give some more details Chromium version and system would help this seems completely off did you have the latest update?

**Comment by ghost** [comment]


**Reply by anapnoe** [reply]
it has been fixed you can revert back to your favorite theme on windows best regards

**Comment by anapnoe** [comment]
right click on your desktop personalize settings colors  in the drop down choose your color select dark
reload the ui-ux I will fix it thanks

**Comment by Nacurutu** [comment]
@anapnoe maybe you should activate the issues tab in the options, same as you activated the discussions. so people can put their issues there. More organized IMO.

**Reply by anapnoe** [reply]
Just open the issues thanks Nacurutu

**Reply by PowerfulShell** [reply]
and simplify the issue tab, dont ask too much :)

**Comment by gmorks** [comment]
just want to tell you @anapnoe I hope this get on the master branch, It should be default thank you for your work

**Reply by anapnoe** [reply]
thank you @gmorks I hope so too

**Comment by foggyflute** [comment]
I just install it today via reddit suggestion, all the basic features that I would want to use on mobile work great so far, thanks for your work.

**Reply by anapnoe** [reply]
you are welcome

**Comment by Skuwwy** [comment]
This is hands down the best A1111 fork I've seen. Excellent work!

**Reply by anapnoe** [reply]
Thank you @Skuwwy

**Comment by notfood** [comment]
I'm liking it so far, my only gripe is that the extension I used to help me change the output size doesn't work here.

https://github.com/thomasasfk/sd-webui-aspect-ratio-helper

**Reply by anapnoe** [reply]
if the sliders are not updated maybe this will help go to settings > user interface and disable the `Dispatch event change on release, for slider and input number components `

**Reply by notfood** [reply]
I did a pull and checked again and several extensions have the same issue, strangely aspect radio is working now after changing `Dispatch event change on release, for slider and input number components`. Could it be a me issue?

Multidiffusion:
```cs
ValueError: File cannot be fetched: extensions/multidiffusion-upscaler-for-automatic1111/javascript/bboxHint.js. All files must contained within the Gradio python app working directory, or be a temp file created by the Gradio python app.
API error: GET: http://127.0.0.1:7860/file=extensions/sd-webui-controlnet/javascript/hints.js?1682390746.5827916 {'error': 'ValueError', 'detail': '', 'body': '', 'errors': 'File cannot be fetched: extensions/sd-webui-controlnet/javascript/hints.js. All files must contained within the Gradio python app working directory, or be a temp file created by the Gradio python app.'
```

TagComplete:
```cs
ValueError: File cannot be fetched: extensions/a1111-sd-webui-tagcomplete/javascript/tagAutocomplete.js. All files must contained within the Gradio python app working directory, or be a temp file created by the Gradio python app.
API error: GET: http://127.0.0.1:7860/file=extensions/multidiffusion-upscaler-for-automatic1111/javascript/bboxHint.js?1682390691.6526027 {'error': 'ValueError', 'detail': '', 'body': '', 'errors': 'File cannot be fetched: extensions/multidiffusion-upscaler-for-automatic1111/javascript/bboxHint.js. All files must contained within the Gradio python app working directory, or be a temp file created by the Gradio python app.'
```

ControlNet:
```cs
ValueError: File cannot be fetched: extensions/sd-webui-controlnet/javascript/hints.js. All files must contained within the Gradio python app working directory, or be a temp file created by the Gradio python app.
```

**Reply by anapnoe** [reply]
try restart the terminal I only checked tagcomplete and there are no errors make sure you have the latest ui-ux commit
also you can open issues

**Comment by jurandfantom** [comment]
Hey Anapnoe,
mega thanks for make effort with fork and all time you spend with tinkering and fixing. Hopefully everything gonna work (compare to Vlad ) :) List of things that works is impressive, but to be honest, as hardcore user of automatic1111 (since beginning of SD ) im more interested in information of what NOT working compare to A1111 - list should be shorter and contain very essential information for those (like me ) who are looking for some backup or perhaps, new home.

Will instal your fork tomorrow and start digging for bugs and issues :) fingers cross everything goes well with your project.

**Reply by anapnoe** [reply]
mega welcome jurandfantom 
@vladmandic is working hard to make things stable and he is very capable of doing so we are collaborating to release a new version as a new base and we are hoping more people to join in as a collective to move things forward

**Reply by Nacurutu** [reply]
@anapnoe waiting for that collab, your fixes + UX/UI + Vlad's fixes and improvements would be so great..!!!
